src/screens/Home/Home.js

```diff
@@ -337,22 +332,6 @@ class HomeScreen extends React.Component<Props, State> {
     return !isEq;
   }
 
-  validateWalletConnectQRCode = (uri: string) => {
```

Comment:
Are you sure all the deletions (according to this git diff) in this file should be deleted?

Reply:
These are being moved into the `MeScreen`